--- origsrc/units-1.87/Makefile.in	2007-09-26 20:06:31.000000000 -0500
+++ src/units-1.87/Makefile.in	2009-06-22 03:40:23.186433500 -0500
@@ -77,13 +77,13 @@ parse.tab.@OBJEXT@: parse.tab.c
 units@EXEEXT@: $(OBJECTS)
 	$(CC) $(LDFLAGS) -o units $(OBJECTS) $(LIBS)
 
-install: units units.dat install-doc
+install: units@EXEEXT@ units.dat install-doc
 	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(datadir)
-	$(INSTALL_PROGRAM) units $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'`
+	$(INSTALL_PROGRAM) units@EXEEXT@ $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'`@EXEEXT@
 	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/units.dat
 
-install-strip: units units.dat install-doc
-	$(INSTALL_PROGRAM) -s units $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'`
+install-strip: units@EXEEXT@ units.dat install-doc
+	$(INSTALL_PROGRAM) -s units@EXEEXT@ $(DESTDIR)$(bindir)/`echo units|sed '$(transform)'`@EXEEXT@
 	$(INSTALL_DATA) $(srcdir)/units.dat $(DESTDIR)$(datadir)/units.dat
 
 install-doc: install-man install-info
