--- origsrc/units-2.01/Makefile.in	2012-08-07 19:41:02.000000000 -0500
+++ src/units-2.01/Makefile.in	2013-07-08 01:31:18.536369700 -0500
@@ -81,7 +81,7 @@ units@EXEEXT@: $(OBJECTS)
 	$(CC) $(CFLAGS) $(LDFLAGS) -o units@EXEEXT@ $(OBJECTS) $(LIBS)
 
 units_cur_inst: units_cur
-	sed -e "s:outfile = 'currency.units':outfile='@UDAT@currency.units':" units_cur > units_cur_inst
+	sed -e "s:outfile = 'currency.units':outfile='@UDAT@currency.units':" $< > units_cur_inst
 
 install-support: definitions.units currency.units units_cur_inst
 	$(srcdir)/mkinstalldirs $(DESTDIR)@UDAT@ $(DESTDIR)$(bindir) 
@@ -158,7 +158,7 @@ maintainer-clean: clean
                units-*.tar.gz parse.tab.c currency.units
 
 currency.units: 
-	./units_cur > currency.units
+	cd $(srcdir) && ./units_cur
 
 sig:
 	echo units-`sed -n -e '/#.*VERSION/s/.*"\(.*\)"/\1/gp' \
